### ISSUE TITLE
1.6.0: tp.frontmatter returns "NaN" instead of string

### ISSUE THREAD

**Issue by Moonbase59** [SilentVoid13/Templater#195]
**Plugin informations (please complete the following information):**
 - OS: Linux Mint 20.1/Cinnamon
 - Templater version: 1.6.0
 - Obsidian version: 0.11.13/AppImage, Installer 0.10.11
 - Templater settings: –

**Describe the bug**
`<%+ tp.frontmatter.relation %>` returns `NaN` instead of "Tochter".

**Expected behavior**
Should return the value in the front matter, in this case "Tochter"

**Screenshots**
![obsidian-templater-bug-NaN](https://user-images.githubusercontent.com/3706922/117245332-fc599e80-ae3a-11eb-85d3-f43e18314083.png)

**Additional context**

My front matter:
```
---
location: [51.133333,10.416667]
mapmarker: Family
birthday: 1997-05-17
deceased: 2020-05-17
relation: Tochter
tags: family
distance: 100.0
---
```

My code in the note, comparing _Dataview_ and _Templater_:

```
\* `= this.birthday` (`= (date(today) - this.birthday).years + " Jahre"`) † `= this.deceased` `= this.relation`

\* <%+ moment(tp.frontmatter.birthday).format("dd., YYYY-MM-DD") %> (<%+ moment().diff(moment(tp.frontmatter.birthday), "years") %> Jahre) † <%* if (tp.frontmatter.deceased) {tR+=moment(tp.frontmatter.deceased).format("dd., YYYY-MM-DD")} else {tR+="-"} %> <%+ tp.frontmatter.relation %>
```

_Note:_ Data is fake, my daughter happily lives. This was just put in for testing.

**Comment by Moonbase59** [comment]
Probably related: https://github.com/SilentVoid13/Templater/issues/188

**Comment by Moonbase59** [comment]
With 1.7.0, above now returns `<%+ tp.frontmatter.relation %>` instead of `NaN` or `Tochter` (as it should).

**Comment by SilentVoid13** [comment]
Hey @Moonbase59, sorry for the late response. Thanks for the report. This should be fixed now as of Templater 1.7.1.

Update and tell me if this works now.

**Comment by Moonbase59** [comment]
@SilentVoid13 No problem, your response times are great! 1.7.1 fixed it, thanks!
